QModManager/API/SMLHelper/Patchers/CraftTreePatcher.cs

```diff
@@ -0,0 +1,231 @@
+﻿namespace QModManager.API.SMLHelper.Patchers
+{
+    using Harmony;
+    using System.Collections.Generic;
+    using System.Reflection;
+    using Utility;
+    using Crafting;
+    using System;
+
+    internal static class CraftTreePatcher
+    {
+        #region Internal Fields
+
+        internal static Dictionary<CraftTree.Type, ModCraftTreeRoot> CustomTrees = new Dictionary<CraftTree.Type, ModCraftTreeRoot>();
+        internal static List<Node> NodesToRemove = new List<Node>();
+        internal static List<CraftingNode> CraftingNodes = new List<CraftingNode>();
+        internal static List<TabNode> TabNodes = new List<TabNode>();
+
+        #endregion
+
+        #region Patches
+
+        internal static void Patch(HarmonyInstance harmony)
+        {
+            harmony.Patch(AccessTools.Method(typeof(CraftTree), "GetTree"),
+                prefix: new HarmonyMethod(AccessTools.Method(typeof(CraftTreePatcher), "GetTreePreFix")));
+
+            harmony.Patch(AccessTools.Method(typeof(CraftTree), "Initialize"),
+                postfix: new HarmonyMethod(AccessTools.Method(typeof(CraftTreePatcher), "InitializePostFix")));
+
+            harmony.Patch(AccessTools.Method(typeof(CraftTree), "FabricatorScheme"),
+                postfix: new HarmonyMethod(AccessTools.Method(typeof(CraftTreePatcher), "FabricatorSchemePostfix")));
```

Comment:
Easily accessible using publicized assemblies

Reply:
Probably not since these aren't instances of accessing a field or property but rather getting a `MethodInfo` for Harmony patching.